main.py

```diff
@@ -155,60 +164,58 @@ def get_interfaces(self):
 
         return jsonify({'interfaces': interfaces})
 
-    @rest('v3/interfaces/enable', methods=['POST'])
+    @rest('v3/interfaces/switch/<dpid>/enable', methods=['POST'])
     @rest('v3/interfaces/<interface_enable_id>/enable', methods=['POST'])
-    def enable_interface(self, interface_enable_id=None):
+    def enable_interface(self, interface_enable_id=None, dpid=None):
         """Administratively enable a list of interfaces in the topology."""
         error_list = []  # List of interfaces that were not activated.
         interface_ids = []
         msg_error = "Some interfaces couldn't be found and activated: "
```

Comment:
```suggestion
        error_list = []  # List of interfaces that were not activated.
        msg_error = "Some interfaces couldn't be found and activated: "
```

You don't need to keep a list of interfaces.